components/reusable-components/footer.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Link from 'next/link';
+import { FaLinkedin, FaFacebook, FaInstagram, FaYoutube } from 'react-icons/fa';
+import styles from './footer.module.scss';
+
+function Footer() {
+  return (
+    <footer className={styles.footer}>
+      <div className={styles.leftSection}>
+        <p>LearnHub.mk</p>
+      </div>
+
+      <div className={styles.contactSection}>
+        <p>CONTACT</p>
+        <p>contact@learnhub.mk</p>
+      </div>
+
+      <div className={styles.socialMediaSection}>
+        <p>SOCIAL MEDIA</p>
+        <Link href="https://www.linkedin.com/" target="_blank" rel="noopener noreferrer">
+          <FaLinkedin /> LINKEDIN
+        </Link>
```

Comment:
All Social Media Links are uppercase. You can use text-transform: uppercase to achieve that.